src/Display/Tabs/Tabs.tsx

```diff
@@ -10,7 +10,7 @@ import {
   TabsBody,
 } from '../../Style/Display/TabsStyle';
 
-const Tabs: Tabs = ({activeTab, onTabClick, children, className}) => {
+const Tabs: Tabs = ({activeTab, onTabClick, children, variant, className, alignment}) => {
```

Comment:
Remember to set the `default props` for `variant` and `alignment`. Realised I need to update storybook for the `activeTab` and `onTabClick` prop too haha forgot about that. But no worries, I'll wait for your PR to be merged before updating.